tests/client/test_client.py

```diff
@@ -156,6 +155,29 @@ def test_lit_references(self):
         response = client.lit.references(self.db_keys.assessment_client)
         assert isinstance(response, pd.DataFrame)
 
+    def test_lit_reference(self):
```

Comment:
@rabstejnek minor rewrite of the test (mostly to fix the api), but I think it reads well too, what do you think?

Reply:
I like the changes!